DisplayCAL/wxwindows.py

```diff
@@ -5236,19 +5236,18 @@ def fancytext_Renderer_getCurrentColor(self):
 fancytext.Renderer.getCurrentFont = fancytext_Renderer_getCurrentFont
 
 
-def fancytext_RenderToRenderer(str, renderer, enclose=True):
-    str = safe_str(str, "UTF-8")
```

Comment:
Let me fix it in this way, I replaced with ``str_`` but ``text`` seems better :+1:

Reply:
thank you for doing the merge